src/lxc/utils.c

```diff
@@ -910,9 +910,24 @@ char *get_template_path(const char *t)
 	int ret, len;
 	char *tpath;
 
-	if (t[0] == '/' && access(t, X_OK) == 0) {
-		tpath = strdup(t);
-		return tpath;
+	if (t[0] == '/') {
+        	if (access(t, X_OK) == 0) {
+			tpath = strdup(t);
+			return tpath;
```

Comment:
Just do `return strdup(tpath)` :)

Reply:
As I mentionned in the commit comment, the original code does not return iommediately upon this error and tries to open ".../lxc-/pathname_passed". So, the original errno is lost.
But you are right, it is not necessary to dicriminate the errnos in the "else" case.
Concerning "tpath =", I kept the original :-)